### PR TITLE
[KC-2018] Add Kansas City 2018

### PR DESCRIPTION
**Pull request by barkerd427**
Our team is going to review this before we merge it in.

### CODE REVIEW

File: data/events/2018-kansas-city.yml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Aaron Blythe"
+    twitter: "ablythe"
+    employer: "Hearst Business Media"
```
Comment (aaronblythe) [inline data/events/2018-kansas-city.yml]: New Relic

File: data/events/2018-kansas-city.yml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    bio: "father, husband, hacker, software architect, genuinely interested in understanding things and making them better."
+    image: aaron-blythe.jpg
+    github: aaronblythe
+    website: https://aaronblythe.org
+    linkedin: https://www.linkedin.com/in/aaronblythe/
+  - name: "Dan Barker"
+    twitter: "barkerd427"
+    employer: "DST Systems"
```
Comment (aaronblythe) [inline data/events/2018-kansas-city.yml]: NAIC

File: data/events/2018-kansas-city.yml [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  - name: "Rebecca Sowers"
+    image: rebecca-sowers.jpg
+    employer: Cerner Corporation
+    linkedin: https://www.linkedin.com/in/rebecca-sowers-931a8842/
+    bio: "Rebecca is currently a Service Portfolio Manager focusing on tools that monitor, automate, configure, and discover all layers of infrastructure. In her free time, she loves to do home renovation projects in her old 1903 \"money pit\"."
+  - name: "Dustin Holub"
+    employer: Technology Group Solutions
+    # image: dustin-holub.jpg
+    linkedin: https://www.linkedin.com/in/dustinholub/
+    bio: ""
```
Comment (aaronblythe) [inline data/events/2018-kansas-city.yml]: Bio from Linkedin: 

I'm an outgoing individual who believes that work and fun can be one in the same. I strive to help those around me realize what can be achieved through application of technology in both professional and personal lives.

File: data/events/2018-kansas-city.yml [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    linkedin: https://www.linkedin.com/in/bertweidt/
+    bio: "Systems Architect and Security guy, but involved in many aspects of Technology for clients of all sizes. DevOps has changed the world for how solutions are implemented both internally and delivered to clients globally. Looking forward to what's next, other than self-driving cars. DevOpsDays is a great forum for sharing what's working, and talking about that thing that everyone wants most - innovation."
+  - name: "Rebecca Sowers"
+    image: rebecca-sowers.jpg
+    employer: Cerner Corporation
+    linkedin: https://www.linkedin.com/in/rebecca-sowers-931a8842/
+    bio: "Rebecca is currently a Service Portfolio Manager focusing on tools that monitor, automate, configure, and discover all layers of infrastructure. In her free time, she loves to do home renovation projects in her old 1903 \"money pit\"."
+  - name: "Dustin Holub"
+    employer: Technology Group Solutions
+    # image: dustin-holub.jpg
```
Comment (aaronblythe) [inline data/events/2018-kansas-city.yml]: Image from LinkedIn: https://media-exp2.licdn.com/media/AAEAAQAAAAAAAAxmAAAAJDZiZTNjY2Q1LTczYTEtNGFiMy1iNDkzLTY0ODJmYzU5Y2Q2Nw.jpg

File: content/events/2018-kansas-city/sponsor.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
++++
+Title = "Sponsor"
+Type = "event"
+Description = "Interested in sponsoring DevOpsDays Kansas City 2017? We greatly value sponsors for this open event."
++++
+
+<div class = "row">
+<div class = "col-md-8 col-sm-12">
+We greatly value sponsors for this community event. If you are interested in sponsoring, please check out our <a href="/events/2017-kansascity/DODKC17_Prospectus.jpg">prospectus</a> or <a href="mailto:organizers-kansascity-2018@devopsdays.org?subject=DevOpsDays%20Kansas%20City%202018%20Sponsorship">send us an email</a>.
```
Comment (aaronblythe) [inline content/events/2018-kansas-city/sponsor.md]: Need to update jpg Prospectus to a 2018 prospectus.

File: content/events/2018-kansas-city/sponsor.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+<tr><td>What should we expect regarding electricity? (how much, any fees, etc)</td><td></td></tr>
+<tr><td>What should we expect regarding WiFi? (how much, any fees, etc)</td><td></td></tr>
+<tr><td>How do we order additional A/V equipment?</td><td></td></tr>
+<tr><td>Additional important details</td><td></td></tr>
+</table> -->
+</div>
+</div>
+</div>
+<div class = "col-md-4 col-sm-12">
+<a href = "/events/2017-kansascity/DODKC17_Prospectus.jpg"><img src = "/events/2017-kansascity/DODKC17_Prospectus_small.jpg" class="img-fluid""></a>
```
Comment (aaronblythe) [inline content/events/2018-kansas-city/sponsor.md]: Need to update link to a 2018 prospectus.

**Comment by bksteiny** [comment]
Any reason Caleb's picture is included in this PR?

File: data/events/2018-kansas-city.yml [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+# You may optionally include a "max" attribute to limit the number of sponsors per level. For
+# unlimited sponsors, omit the max attribute or set it to 0. If you want to prevent all
+# sponsorship for a specific level, it is best to remove the level.
+sponsor_levels:
+  - id: ultimate
+    label: Ultimate
+    max: 1
+  - id: premium
+    label: Premium
+    max: 0 # This is the same as omitting the max limit.
```
Comment (bksteiny) [inline data/events/2018-kansas-city.yml]: 🤔 

Would we put a cap on this later or keep it as is and see what happens?

File: content/events/2018-kansas-city/sponsor.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+<p>
+Sponsors are responsible for providing high-res logos for use on the website and in promotional materials.  The preferred file formats are EPS or AI.
+
+<hr/>
+<div class = "row">
+<div class = "col-sm-12">
+  <table class = "table table-bordered table-responsive">
+  <thead>
+    <tr>
+      <th><i>packages</i></th>
```
Comment (bksteiny) [inline content/events/2018-kansas-city/sponsor.md]: I like the tiering 👍

**Comment by MikeRosTX** [comment]
@barkerd427 let us know when changes have been applied so we can proceed with the merge.

**Comment by barkerd427** [comment]
@MikeRosTX, will do. I'll change the subject and post here and slack.

File: data/events/2018-kansas-city.yml (hunk 7 above)
Comment (barkerd427) [reply data/events/2018-kansas-city.yml]: I'll cap it at 10.

File: content/events/2018-kansas-city/sponsor.md (hunk 5 above)
Comment (barkerd427) [reply content/events/2018-kansas-city/sponsor.md]: Working on that now. I may just remove it initially.

**Comment by barkerd427** [comment]
I'll remove Caleb's picture. That was a copy/paste error.

File: data/events/2018-kansas-city.yml (hunk 4 above)
Comment (ThePVD) [reply data/events/2018-kansas-city.yml]: Slightly better cropping / res here:  https://drive.google.com/file/d/1HJiBr_gv2zO4XU1vLD9dIbuYWbdpPU3Z/view?usp=sharing  ...hoping to get a new shot taken this year.

File: data/events/2018-kansas-city.yml (hunk 3 above)
Comment (ThePVD) [reply data/events/2018-kansas-city.yml]: Updating to 3rd person / tweaks:

"Dustin is an outgoing individual who believes that work and fun can be one in the same. After spending many years of his career in infrastructure silos, he was drawn to the DevOps culture and its emphasis on learning and solving problems together.  Dustin spends his free time chasing his two small children, playing indoor soccer, and wondering when technology will completely free him from housework."

File: data/events/2018-kansas-city.yml (hunk 4 above)
Comment (barkerd427) [reply data/events/2018-kansas-city.yml]: @ThePVD, I need it to be square.

File: data/events/2018-kansas-city.yml (hunk 1 above)
Comment (barkerd427) [reply data/events/2018-kansas-city.yml]: Done.

File: data/events/2018-kansas-city.yml (hunk 2 above)
Comment (barkerd427) [reply data/events/2018-kansas-city.yml]: Done.

File: data/events/2018-kansas-city.yml (hunk 3 above)
Comment (barkerd427) [reply data/events/2018-kansas-city.yml]: Done.

File: data/events/2018-kansas-city.yml (hunk 7 above)
Comment (barkerd427) [reply data/events/2018-kansas-city.yml]: Done.

**Comment by barkerd427** [comment]
I'm still waiting on a couple more items before we merge.

File: data/events/2018-kansas-city.yml [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+  - name: "Rebecca Sowers"
+    image: rebecca-sowers.jpg
+    employer: Cerner Corporation
+    linkedin: https://www.linkedin.com/in/rebecca-sowers-931a8842/
+    bio: "Rebecca is currently a Service Portfolio Manager focusing on tools that monitor, automate, configure, 
+    and discover all layers of infrastructure. In her free time, she loves to do home renovation projects in her 
+    old 1903 \"money pit\"."
+  - name: "Dustin Holub"
+    employer: Technology Group Solutions
+    # image: dustin-holub.jpg
```
Comment (ThePVD) [inline data/events/2018-kansas-city.yml]: updated 1:1 source: https://drive.google.com/open?id=1HJiBr_gv2zO4XU1vLD9dIbuYWbdpPU3Z

Comment (barkerd427) [reply data/events/2018-kansas-city.yml]: Added.

**Comment by bridgetkromhout** [comment]
Hi, @barkerd427 - how are we doing on this? Getting close to being able to merge? Thanks.

**Comment by barkerd427** [comment]
Work has gotten to be too much this week, and I haven't been able to get back to this. I have a few changes to make and then it's ready. I may not be able to get that done until Friday given my current schedule.

**Comment by bridgetkromhout** [comment]
Hey @barkerd427, glad to hear you're close! Let's get this one merged this weekend and you can always iterate!

**Comment by barkerd427** [comment]
Definitely. I just need to find a little time on my laptop. Probably
tonight.

**Comment by barkerd427** [comment]
@bridgetkromhout, this one should do it. I made the suggested changes and backed out a couple others I didn't have time to finish.

**Comment by barkerd427** [comment]
Not sure why that failed.

**Comment by barkerd427** [comment]
I think I fixed it and updated it for the new Hugo stuff.

**Comment by mattstratton** [comment]
For future reference, if you are going to need to have approvals outside of the core team before merges, I recommend using your own repo (with netlify!) to generate previews for the rest of your team to check out.

I wrote something up here - hopefully it works for you https://github.com/devopsdays/devopsdays-web/tree/master/utilities/docs/workflow

**Comment by barkerd427** [comment]
Thank you!

**Comment by bridgetkromhout** [comment]
Okay, merging! Thanks!

**Pull request merged**